### PR TITLE
Mostly Arabic updates

### PR DESCRIPTION
**Pull request by Link4K**

### CODE REVIEW

File: channels/sa.m3u [hunk 1]
```diff
@@ -17,10 +17,12 @@ http://m.live.net.sa:1935/m/tanasuhtv/playlist.m3u8
 http://ayat.live.net.sa:1935/live/ayat/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/7aD1yan.jpg" group-title="",Beity 
 http://82.212.74.99:8000/live/8164.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/KRuV5yv.png" group-title="",CITRUSS 
-https://citrusstv.akamaized.net/hls/live/687285/CTV/index.m3u8
```
Comment (freearhey) [inline channels/sa.m3u]: This channel works for me. (Tested in VLC)
Captura de pantalla 2020-02-07 a las 8 27 20

**Review by freearhey — CHANGES_REQUESTED** [review]
Great update! Please return the CITRUSS channel back so we can approve this update.

File: channels/sa.m3u (hunk 1 above)
Comment (Link4K) [reply channels/sa.m3u]: It's a duplicate of https://github.com/iptv-org/iptv/blob/master/channels/ae.m3u#L48

Comment (freearhey) [reply channels/sa.m3u]: No, it's not. In fact, it leads to HLS playlist that contains several broadcast options in different quality, including your option.

Captura de pantalla 2020-02-08 a las 10 05 26

Comment (Link4K) [reply channels/sa.m3u]: added back

**Review by freearhey — APPROVED** [review]

**Pull request merged**